---
title: "Background knowledge for the Tidy Transcriptomics workshop"
author:
  - Maria Doyle, Peter MacCallum Cancer Centre^[maria.doyle@petermac.org]
  - Stefano Mangiola, Walter and Eliza Hall Institute^[mangiola.s@wehi.edu.au]
output: rmarkdown::html_vignette
vignette: >
  %\VignetteIndexEntry{Background knowledge for the Tidy Transcriptomics workshop}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
---

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>"
)
```

```{r, echo=FALSE, out.width = "100px"}
knitr::include_graphics("../inst/vignettes/tidybulk_logo.png")
```

# Schedule

- Part 1 Bulk RNA-seq Core
- Part 2 Bulk RNA-seq Extended
- Part 3 Single-cell RNA-seq

A more detailed schedule can be found [here](https://stemangiola.github.io/biocasia2020_tidytranscriptomics/index.html#time-outline).  
<br>
**Format**: a few introductory slides, then demos, exercises plus Q&A.  
**Interact**: Zoom chat and polls
<br>
<br>

# What is transcriptomics?

“The transcriptome is the set of all RNA transcripts, including coding and non-coding, in an individual or a population of cells”

_Wikipedia_

```{r, echo=FALSE, out.width = "600px"}
knitr::include_graphics("../inst/vignettes/transcriptomics.jpg")
```

# Why use transcriptomics?

- Genome (DNA) pretty stable
- Proteome (proteins) harder to measure
- Transcriptome (RNA) can measure changes in expression of thousands of coding and non-coding transcripts
<br>
<br>
```{r, echo=FALSE, out.width = "600px"}
knitr::include_graphics("../inst/vignettes/ScreenShot1.png")
```

# Possible experimental design

```{r, echo=FALSE, out.width = "600px"}
knitr::include_graphics("../inst/vignettes/ScreenShot2.png")
```

# How does transcriptomics work?

```{r, echo=FALSE, out.width = "600px"}
knitr::include_graphics("../inst/vignettes/ScreenShot3.png")
```

# Types of transcriptomic analyses

* **Differential expression**  
* **Cell type composition**  
* Alternative splicing  
* Novel transcript discovery  
* Fusions identification  
* Variant analysis  
<br>
Topics in bold we will see in this workshop
<br>
<br>

# Bulk RNA sequencing differential expression workflow

```{r, echo=FALSE, out.width = "600px"}
knitr::include_graphics("../inst/vignettes/biocasia2020pipeline-01.png")
```

# Differences between bulk and single-cell RNA sequencing

```{r, echo=FALSE, out.width = "600px"}
knitr::include_graphics("../inst/vignettes/bulk_vs_single.jpg")
```

_Shalek and Benson, 2017_

# Single-cell RNA sequencing analysis workflow

```{r, echo=FALSE, out.width = "600px"}
knitr::include_graphics("../inst/vignettes/biocasia2020pipeline-02.png")
```

# Getting started 

## Cloud

Easiest way to run this material. Only available during workshop.  
 * You should have received an invite to join the classroom in the R in Pharma Organization space in RStudio Cloud.   
 * Login and click the "Start" button as shown in the screenshot below  
 * Open `tidytranscriptomics.Rmd` in `vignettes` folder
 
```{r, echo=FALSE, out.width = "600px"}
knitr::include_graphics("../inst/vignettes/rstudio_cloud.png")
```
 


## Local

If you want to install on your own computer, see instructions [here](https://stemangiola.github.io/biocasia2020_tidytranscriptomics/index.html#workshop-package-installation). We recommend using the Cloud during the biocasia workshop and this method if you want to run the material after the workshop.

